apidoc/Titanium/UI/SearchBar.yml

```diff
@@ -165,11 +165,28 @@ properties:
     since: "5.4.0"
     platforms: [iphone,ipad]
 
+  - name: color
+    summary: Color of the text in this text field, as a color name or hex triplet.
+    description: |
+        For information about color values, see the "Colors" section of <Titanium.UI>.
+    type: String
+    platforms: [iphone, ipad]
```

Comment:
Android already supports this, just wasn't documented. Would you mind adding "android" to the supported platform list please?